content/healthchecks/livenessprobe.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Configure Liveness Probe"
+chapter: true
+weight: 5
+---
+
+# Configure Liveness Probe
+
+```
+kubectl apply -f ~/environment/healthchecks/liveness-app.yaml
```

Comment:
Can you go through the details of how liveness is configured? and highlight when you kill nodejs application what happens under the covers with Kubernetes?